packages/core/src/components/time-picker/time-picker.tsx

```diff
@@ -366,9 +373,9 @@ export class TimePicker {
           });
 
           if (this.focusScrollAlignment === 'end') {
-            elementList.scrollTop += 4;
+            elementList.scrollTop += 3;
```

Comment:
Not sure if related to this change but 00 will sit a little above the other numbers:
<img width="184" alt="Screenshot 2025-06-18 at 13 57 27" src="https://github.com/user-attachments/assets/98291262-62e2-46ce-89a9-3bc1ee748ae0" />